Dockerfile

```diff
@@ -1,5 +1,7 @@
 # Build node feature discovery
-FROM golang:1.8 as builder
+FROM golang:1.11-alpine as builder
+
+RUN apk add --no-cache gcc git libc-dev
```

Comment:
As we do `RUN make test` instead of `RUN go test` since #209, we must install `make` too. Otherwise it seems to be working as expected.